app/models/school.rb

```diff
@@ -104,6 +104,21 @@ def preorder_status_or_default
     end
   end
 
+  def perceived_state
```

Comment:
not sure about this one. The state in the database seems fixed and we kind of want a virtual one. Any better ideas would be welcomed

Reply:
i'm not convinced about the need to have a separate status. I think it'd be much simpler to extend `PreorderInformation#infer_status` with some of this logic and update `PreorderInformation#status` appropriately.

Reply:
We'd need to recalculate some existing statuses in the DB but that's OK...

Reply:
I've added an extra commit that melds `perceived_state` into `PreorderInformation#infer_status`